GPU/GPUTracking/CMakeLists.txt

```diff
@@ -262,18 +266,26 @@ unset(HDRS_TMP)
 
 # Main CMake part for O2
 if(ALIGPU_BUILD_TYPE STREQUAL "O2")
+
   o2_add_library(GPUDataTypes
-                 TARGETVARNAME targetName
-                 PUBLIC_INCLUDE_DIRECTORIES .
-                                            Definitions
-                                            DataTypes
-                 PUBLIC_LINK_LIBRARIES O2::GPUUtils
-                                       O2::GPUCommon
-                                       O2::ReconstructionDataFormats
-                                       O2::TPCFastTransformation
-                 PRIVATE_LINK_LIBRARIES O2::DataFormatsTPC
-                 SOURCES ${SRCS_DATATYPES})
-  target_compile_definitions(${targetName} PRIVATE GPUCA_O2_LIB GPUCA_TPC_GEOMETRY_O2)
+                  TARGETVARNAME targetName
+                  PUBLIC_INCLUDE_DIRECTORIES .
+                                              Definitions
+                                              DataTypes
+                  PUBLIC_LINK_LIBRARIES O2::GPUUtils
+                                        O2::GPUCommon
+                                        O2::ReconstructionDataFormats
+                                        O2::TPCFastTransformation
+                  PRIVATE_LINK_LIBRARIES O2::DataFormatsTPC
+                  SOURCES ${SRCS_DATATYPES})
+  if(NOT ALIGPU_BUILD_TYPE STREQUAL "Standalone")
```

Comment:
I am sorry for my previous comment here. This is actually not needed to move ML down, since we are in a `if(ALIGPU_BUILD_TYPE STREQUAL "O2")` branch, so we don't need the check `if(NOT ALIGPU_BUILD_TYPE STREQUAL "Standalone")`